src/ui/screens/mainwindow/mainwindowstatusbar.cpp

```diff
@@ -96,6 +103,9 @@ namespace tremotesf {
         QObject::connect(mRpc->serverStats(), &ServerStats::updated, this, [=, this] {
             mDownloadSpeedLabel->setText(formatutils::formatByteSpeed(mRpc->serverStats()->downloadSpeed()));
             mUploadSpeedLabel->setText(formatutils::formatByteSpeed(mRpc->serverStats()->uploadSpeed()));
+            mFreeSpaceLabel->setText(
+                QObject::tr("Free space: %1").arg(formatutils::formatByteSize(mRpc->serverStats()->freeSpace()))
```

Comment:
`qApp->translate("tremotesf", "string")` is used throughout the project instead of `QObject::tr()`